<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageAccountWidget</class>
 <widget class="QWidget" name="ManageAccountWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>578</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="vlay">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Incoming accounts (add at least one):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mFilterAccount">
       <property name="placeholderText">
        <string>Filterâ€¦</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hlay">
       <item>
        <widget class="Akonadi::AgentInstanceWidget" name="mAccountList"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="btn_vlay">
         <item>
          <widget class="QPushButton" name="mAddAccountButton">
           <property name="text">
            <string>A&amp;dd...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mModifyAccountButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&amp;Modify...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mRemoveAccountButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>R&amp;emove</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mRestartAccountButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Restart</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="0"/>
 <customwidgets>
  <customwidget>
   <class>Akonadi::AgentInstanceWidget</class>
   <extends>QWidget</extends>
   <header>agentinstancewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
